modules/3d/include/opencv2/3d.hpp

```diff
@@ -2596,7 +2618,16 @@ class CV_EXPORTS Octree {
     */
     bool deletePoint(const Point3f& point);
 
-    /** @brief Radius Nearest Neighbor Search in Octree
+    /** @brief restore point cloud data from Octree.
+    *
+    * Restore the point cloud data from existing octree. The points in same leaf node will be seen as the same point.
+    * This point is the center of the leaf node. If the resolution is small, it will work as a downSampling function.
+    * @param restorePointCloud The output point cloud data.
+    * @param restoreColor The color attribute of point cloud data
+    */
+    void getPointCloudByOctree(std::vector<Point3f> &restorePointCloud, std::vector<Point3f> &restoreColor);
```

Comment:
Should this method calculate an average color for the point in the same leaf?
If not, it should be said explicitly in the docs to the function